sir_complainsalot/Makefile

```diff
@@ -13,7 +13,7 @@ install: venv
 install-ci: venv
 	. env/bin/activate; \
 	pip install wheel ; \
-	pip install ../_delphi_utils_python ;\
+	pip install '../_delphi_utils_python[dev]' ;\
```

Comment:
```suggestion
	pip install ../_delphi_utils_python ;\
```

Reply:
Here too: the linter fails when I took the 'dev' off

https://github.com/cmu-delphi/covidcast-indicators/actions/runs/10285993553/job/28465604873?pr=2014